nemo_aligner/utils/utils.py

```diff
@@ -78,6 +78,8 @@ def load_from_nemo(
     """load a model using nemo checkpoint
     """
     connector = CustomSaveRestoreConnector(load_base_model_only=load_base_model_only)
+    if not restore_path:
```

Comment:
I actually added this since for initializing SD, we get ckpt for VAE and Unet separately, so there is no restore_path, and this line actually initialized the model from paths in the cfg